azure-pipelines.yaml

```diff
@@ -0,0 +1,92 @@
+name: chart-powerdns pipeline
+trigger:
+  branches:
+    include:
+    - refs/tags/*
+pr:
+  branches:
+    include:
+    - master
+resources:
+  repositories:
+  - repository: cnp-library
+    type: github
+    name: hmcts/cnp-azuredevops-libraries
+    endpoint: 'HMCTS Github'
+variables:
+  helmWait: 300
+  helmVersion: 2.13.0
+  serviceConnection: "azurerm-sandbox"
+  aksResourceGroup: "cnp-aks-sandbox-rg"
+  aksCluster: "cnp-aks-sandbox-cluster"
+  acrName: "hmctssandbox"
+
+jobs:
+- job: Validate
+  pool:
+    vmImage: 'Ubuntu 16.04'
+  steps:
+  - template: steps/charts/validate.yaml@cnp-library
+    parameters:
+      chartName: powerdns
+      chartReleaseName: chart-powerdns-ci-test
+      chartNamespace: chart-tests
+      helmVersion: $(helmVersion)
+      serviceConnection: $(serviceConnection)
+      aksResourceGroup: $(aksResourceGroup)
+      aksCluster: $(aksCluster)
+      acrName: $(acrName)
+      helmInstallTimeout: $(helmWait)
+- job: ValidateExternalDB
```

Comment:
Should this go to master or is it just to validate it works for now?

Reply:
I've now disabled those tasks because we obviously don't want to keep the PaaS db around forever.  I'd rather keep the tasks in case we need them again.